rabbitmq/src/messages.rs

```diff
@@ -19,6 +19,7 @@ pub struct JobMessage {
     pub download_start_time: DateTime<Utc>,
     pub start_range: u64,
     pub end_range: u64,
+    pub excluded_workers: Vec<String>,
```

Comment:
Hmm... did we think through how this will work with dynamic scaling?
What I worry about is scenario like this:
1. Excluded workers are calculated, we have workers 1 to 10, excluded workers include `9` and `10`
2. Something causes additional scaling and number of workers grows to 20
3. Job gets dispatched and we execute on 18 ouf of 20.

I don't know if that's possible and I'm not even sure if that's a problem if it happens, just asking for your thoughts on that.

Reply:
For now it doesn't create any problems, jobs are processed sequentially, scaling job is always before first work job, and partial job is always first out of two. Moreover there is a fixed no of workers atm. 
Even if we account for variable max no of workers then partial job still will take around 80% of them, excluding the rest.
This might be a thing if we would implement parallel jobs.
Should I reverse the logic from `exclude` to `include`, naming exact workers that should work on a job?

Reply:
>  Should I reverse the logic from exclude to include, naming exact workers that should work on a job?

Nah, lets keep it like it is now.